manifests/s/sigoden/Dufs/0.31.0/sigoden.Dufs.installer.yaml

```diff
@@ -0,0 +1,24 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS1.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: sigoden.Dufs
+PackageVersion: 0.31.0
+InstallerType: zip
+ReleaseDate: 2022-11-12
+Installers:
```

Comment:
```suggestion
Dependencies:
  PackageDependencies:
    - PackageIdentifier: Microsoft.VCRedist.2015+.x64
Installers:
```

Reply:
Hi @SpecterShell,

The package installs, but gives this message when launched:
![image](https://user-images.githubusercontent.com/114952565/215864974-c243a014-ebd3-4eac-87eb-c390c64196fa.png)